src/reservation.js

```diff
@@ -0,0 +1,31 @@
+function reservation() {
+  const rem = document.querySelector('.section-one');
+  rem.remove();
+
+  const artsection = document.querySelector('.caption-title');
+  const art = document.createElement('article');
+  const heading1 = document.createElement('h1');
+  const firstpara = document.createElement('p');
+  const formm = document.createElement('form');
+  const input1 = document.createElement('input');
+  const input2 = document.createElement('input');
+  const input3 = document.createElement('input');
+
+  // secondpara.classList.add("contactpage");
```

Comment:
## Remove comments
- [ ] Comments: It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

Reply:
Removed comment